custom/icds_reports/reports/awc_reports.py

```diff
@@ -1038,7 +1050,12 @@ def base_data(row_data):
             ),
             pse_days_attended=row_data.pse_days_attended,
             mother_phone_number=row_data.mother_phone_number,
-            aww_phone_number=row_data.aww_phone_number
+            aww_phone_number=row_data.aww_phone_number,
+            beneficiary_status=get_beneficiary_status(row_data.valid_status_daily,
```

Comment:
This is being ignored in the FE yeah for the FF?

Reply:
i have handled it in the UI